src/Core/Command/UploadAvatarHandler.php

```diff
@@ -96,7 +96,10 @@ public function handle(UploadAvatar $command)
         $this->validator->assertValid(['avatar' => $file]);
 
         $manager = new ImageManager;
-        $manager->make($tmpFile)->fit(100, 100)->save();
+
+        $encodedImage = $manager->make($tmpFile)->fit(100, 100)->encode('jpg', 100);
+
+        @file_put_contents($tmpFile, $encodedImage);
```

Comment:
I'm also curious whether that works. Tmp is not one of the identifiable extensions, so the encoding should force during save, see:

http://image.intervention.io/api/encode